src/ir/module-utils.h

```diff
@@ -377,7 +378,7 @@ template<typename T> struct CallGraphPropertyAnalysis {
     bool hasNonDirectCall = false;
   };
 
-  typedef std::map<Function*, T> Map;
+  typedef ordered_map<Function*, T> Map;
```

Comment:
`CallGraphPropertyAnalysis` is used widely so the extra overhead here worries me a little. Worth measuring the memory and speed impact of this PR.